.github/workflows/kuksa_databroker_build.yml

```diff
@@ -32,6 +32,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Query sites
```

Comment:
Seems to be needed - otherwise builds fail

Reply:
"Query sites" is a weird "name"

the "update" might be needed if the base iage for ubuntu:latest at GH is either so old, that referenced packages in the apt database don't exist anymore, of there is not apt db at all

I am currently not sure if the run stuff is executed in a shell, if it is, might be better to to change the run line of "installing JVM" to

```
run: sudo apt update && sudo apt-get install -y default-jre
```

Reply:
The existing list of package information seemed to be outdated in previous builds, see e.g. https://github.com/eclipse/kuksa.val/actions/runs/5053685693/jobs/9067766790

I will put it within the same run